.github/workflows/pull_request_checks.yml

```diff
@@ -0,0 +1,57 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  name_check:
+    name: Pull Request Name Check
+    runs-on: ubuntu-latest
+    env: 
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is okay... right? 😬 [The token rotates and therefore isn't necessarily a secret](https://docs.github.com/en/actions/security-guides/automatic-token-authentication).

Reply:
Why is this needed if the repo and PR are public?

Reply:
By default the action only has access to reading the repo code itself. These actions need the token to make the API calls necessary to github.